tasks/FindMyMates.tex

```diff
@@ -0,0 +1,52 @@
+\section{Find My Mates [Party Host]}
+
+The robot starts in a room and waits for someone to approach it. The guest knows the name of other guests at the party but doesn’t know how these people look like and where the people are. The robot will gather this information for the guest.
+
+
+\subsection{Main Goal}
+The robot provides the location and unique description of 3 guests of the party as required by another guest.
+
+\noindent\textbf{Reward:} 750 points (250 points per guest).
+
+\subsection{Bonus rewards}
+\begin{enumerate}[nosep]
+	\item Robot providing unique description of a person without having asked such information (75 points)
+\end{enumerate}
+
+
+\subsection{Setup}
+\begin{itemize}
+	\item \textbf{Location}: The robot is in a predefined room at least one person that will approach it.
+
+	\item \textbf{People}: 5 people distributed around the house.
+\end{itemize}
+
+\subsection{Additional rules and remarks}
+\begin{enumerate}[nosep]
+	\item \textbf{Deus ex Machina:} Score reduction for requesting human assistance is applied per guest.
+
+	\item \textbf{Calling the operator for the final answer:} 50\% reduction if the robot does not go back autonomously to the operator to report the information.
+	
+	\item \textbf{Penalty:} If the robot talks to an empty room or does not clearly face a person or crowd when talking (e.g. speaks to a wall), -75 points of penalty.
+\end{enumerate}
+
+
+\subsection{Referee instructions}
+
+The referees need to
+\begin{itemize}
+	\item Make sure there every person has a name.
+	\item Make sure people stay roughly in the same place.
+	\item Distribute the people in the rooms.
```

Comment:
Yeah... Nope.

Reply:
What?

Reply:
For consistency depends on the focus. If you're finding people in the house, point 3 stays. Otherwise rules out.

Reply:
The second point can be like,

* Make sure people stay at the designated place.

Then the third point becomes redundant.